Lib/turtle.py

```diff
@@ -854,13 +854,6 @@ def numinput(self, title, prompt, default=None, minval=None, maxval=None):
 ##############################################################################
 
 
-class Terminator (Exception):
-    """Will be raised in TurtleScreen.update, if _RUNNING becomes False.
-
-    This stops execution of a turtle graphics script.
-    Main purpose: use in the Demo-Viewer turtle.Demo.py.
-    """
-    pass
```

Comment:
Does turtle.Demo still exist?  My impression is that the demo run by running turtle.py is in the test functions of turtle.py.  Is Terminator used in turtledemo.py?   If it is, it is likely not tested since the likely use would likely be for a user to stop running.

Reply:
Oh yes. I overlooked it, other files were using this class.I added to the class to the PR